lib/license_finder/core.rb

```diff
@@ -80,10 +81,15 @@ def current_packages
       @scanner.active_packages
     end
 
+    def clear_logs
+      FileUtils.rm config.log_directory if File.directory? config.log_directory
+    end
+
     def options
       {
         logger: logger,
         project_path: config.project_path,
+        log_directory: config.log_directory,
```

Comment:
can we set this directly to what is merged on line 34?

Reply:
Yep, sounds good.